src/map/pc.c

```diff
@@ -534,6 +535,17 @@ void pc_rental_expire(struct map_session_data *sd, int i) {
 			break;
 	}
 
+	pc->setreg(sd, script->add_str("@expire_itemid"), nameid);
+	pc->setreg(sd, script->add_str("@expire_refine"), sd->status.inventory[i].refine);
+	pc->setreg(sd, script->add_str("@expire_identify"), sd->status.inventory[i].identify);
+	pc->setreg(sd, script->add_str("@expire_attribute"), sd->status.inventory[i].attribute);
+	pc->setreg(sd, script->add_str("@expire_bound"), sd->status.inventory[i].bound);
+	// pc->setreg(sd, script->add_str("@expire_uniqueid"), sd->status.inventory[i].unique_id);
```

Comment:
The reason why it doesn't start from 1 is that it also encodes the character ID of the current character at the time of creation of the item. It was implemented to solve an issue where you could have the same unique ID generated twice, if two mapservers were running at the same time (the same character can't be connected to two different map servers at the same time).

```
return ((uint64)sd->status.char_id << 32) | sd->status.uniqueitem_counter++;
```

The upper 32 bits contain the character ID, while the lower 32 contain a sequential number starting from 1 (a separate counter for each character).

If you need to use those in a script (for example if you want to implement this in a plugin, or something), you could still split it in two variables, and store the two halves separately:

```
pc->setreg(sd, script->add_str("@expire_uniqueid_lo"), (int)(sd->status.inventory[i].unique_id&0xffffffff));
pc->setreg(sd, script->add_str("@expire_uniqueid_hi"), (int)((sd->status.inventory[i].unique_id>>32)&0xffffffff));
```